src/mscorlib/src/System/Globalization/CultureData.Windows.cs

```diff
@@ -21,6 +21,11 @@ namespace System.Globalization
 
     internal partial class CultureData
     {
+        internal static void InitGlobalizationInvariantMode()
+        {
+            s_invariantGlobalizationMode = CLRConfig.GetBoolValue(c_InvariantModeConfigSwitch);
```

Comment:
Why would I ever enable this on Windows? To be certain of behavioral compat with my Unix users?